packages/ingest/src/FtpProviderClient.ts

```diff
@@ -139,15 +173,22 @@ class FtpProviderClient {
    * @param {string} path - path to list
    * @returns {Promise}
    */
-  async list(path) {
+  async list(path: string): Promise<ProviderClientListItem[]> {
     const listFn = this._list.bind(this);
     const files = await recursion(listFn, path);
 
     log.info(`${files.length} files were found on ${this.host}`);
 
     // Type 'type' field is required to support recursive file listing, but
     // should not be part of the returned result.
-    return files.map((file) => omit(file, 'type'));
+    return files.map((file) => ({
```

Comment:
Why do we only want these file properties?

Reply:
Those were the only properties that were being returned before, this just made them more explicit, and easier to define in TS.